tgui/packages/tgui/interfaces/NtosNetMonitor.js

```diff
@@ -86,41 +77,23 @@ const MainPage = (props, context) => {
         may prevent you from reenabling them!
       </NoticeBox>
       <Section title="Wireless Connectivity">
-        {ntnetrelays ? (
-          <LabeledList>
-            <LabeledList.Item label="Active NTNet Relays">
-              {ntnetrelays}
-            </LabeledList.Item>
-          </LabeledList>
-        ) : (
-          'No Relays Connected'
-        )}
-      </Section>
-      <Section title="Firewall Configuration">
-        <LabeledList>
-          <LabeledList.Item
-            label="Software Downloads"
-            buttons={
-              <Button
-                icon={config_softwaredownload ? 'power-off' : 'times'}
-                content={config_softwaredownload ? 'ENABLED' : 'DISABLED'}
-                selected={config_softwaredownload}
-                onClick={() => act('toggle_function', { id: '1' })}
-              />
-            }
-          />
-          <LabeledList.Item
-            label="Communication Systems"
+        {ntnetrelays.map((relay) => (
+          <Section
+            key={relay.ref}
+            title={relay.name}
             buttons={
-              <Button
-                icon={config_communication ? 'power-off' : 'times'}
-                content={config_communication ? 'ENABLED' : 'DISABLED'}
-                selected={config_communication}
-                onClick={() => act('toggle_function', { id: '2' })}
+              <Button.Confirm
+                color={relay.is_operational ? 'good' : 'bad'}
```

Comment:
why not just do green and red here

Reply:
we like to use these in case we want tgui themes later

Reply:
ah alright